modules/common/src/main/resources/application.conf

```diff
@@ -0,0 +1,18 @@
+"snowplow": {
```

Comment:
I was thinking about this namespace and wanted to discuss something like: `snowplow.loader` and `snowplow.shredder` for default configs. In future it'd be cool be able to use one single giantic HOCON for all Snowplow apps and in order to do that we need to work harder to not overlap config parameters for different apps.

Reply:
I have no objection, we can do this change. Do you have any other opinion @istreeter ?

Reply:
It's OK for me if you change the namespace.  But I wonder what kind of user would benefit from having a gigantic hocon instead of many hocons?

The namespace would need to be dynamic.  For example, if the pipeline has a good s3 loader and a bad s3 loader.  Or a good elasticsearch loader and a bad elasticsearch loader.  Or a batch loader for redshift plus a batch loader for Glue catalog.  In all cases, a command line option is needed to tell the app which block to use from the gigantic hocon.

If it helps others reading this conversation, then I will spell out how we use the namespace at the moment.  If the user is configuring the app with system properties on the command line, then we accept namespaced or un-namespaced arguments:

```
-Dsnowplow.storage.schema=atomic
or
-Dstorage.schema=atomic
```

If the user is configuring the app with environment variables, then we also accept namespaced or un-namespaced:

```
export CONFIG_FORCE_snowplow_storage_schema=atomic
or
export CONFIG_FORCE_storage_schema=atomic
```

Finally, a user could technically wrap their entire hocon file in a `snowplow` block, although our examples never demonstrate this because there is no real advantage.

Reply:
The more I think about it..... the namespaced block that we take from the user's gigantic hocon should be dynamic (for the reasons given previously), but the namespace that we use inside the `application.conf` file is static not dynamic.  The future algorithm would look something like this:

1. Inspect the `--namespace` cli argument, which defaults to `snowplow`
2. Read the dynamically namespaced block from the user-provided hocon file.
3. For default/missing properties, read the `snowplow`-namespaced block from `application.conf`.

And therefore I don't think it matters what namespace we use inside the `application.conf` file.